### PR TITLE
Add target to cleanup all piplists

### PR DESCRIPTION
**Pull request by tmichalak**
It is possible for a PIP fuzzer to fail after reaching the defined iteration timeout due an incompatible piplist.
This happens when running all the fuzzers for one architecture and then running them for another architecture after just calling ``make clean`` in the ``fuzzers`` directory. The clean target doesn't clean the piplists which can cause problems during the execution of some pip fuzzers.
Namely, the ``todo`` file used in the pip fuzzer, which is generated based on the specific piplist generated for one architecture may not be compatible for another architecture which might not have the same pips.
The symptom is usually that the fuzzer which is set to finish within max 50 iterations, advances to a certain iteration, but later on performs all remaining iterations until 50 without being able to solve any of the remaining PIPs on the todo list, simply because the new architecture doesn't have those PIPs.

This PR adds an explicit clean_piplists target to the Makefile in the ``fuzzers`` directory and also uses that in kokoro's run script.
A separate target was used instead of adding the cleanup to the clean_fuzzers target to avoid forcing the piplist generation for each iteration that is used in pip_loop.mk.

### CODE REVIEW

**Comment by kgugala** [comment]
LGTM

**Comment by mithro** [comment]
While I think this is a good CL I don't think it can be a cause of the failures? Each Kokoro builder is generating a fresh database for exactly one device type?

**Comment by tmichalak** [comment]
@mithro: you are right, this PR doesn't fix the instability.
It simply adds a CL which is very important when running the pip fuzzers locally and switching between architectures. I described above the consequences of not cleaning the piplists when running the 041 fuzzer for another architecture.

**Comment by mithro** [comment]
Please update the description and then I can merge.

**Comment by tmichalak** [comment]
@mithro The description has been updated.

**Comment by mithro** [comment]
@tmichalak I don't see it updated? It still mention that is solving 041-clk-hrow-pips fuzzer reported in #784?

**Comment by tmichalak** [comment]
@mithro: I guess now the description is fine.

**Comment by tmichalak** [comment]
Changed to WIP as it's constantly red.
Despite he fact that it's RED due to the the prjxray stability issues, we can't merge it until it's green.

File: fuzzers/050-pip-seed/Makefile [hunk 1]
```diff
@@ -3,6 +3,9 @@ ifeq ($(QUICK),Y)
 N ?= 1
 SEGMATCH_FLAGS=
 else
+ifeq (${XRAY_DATABASE}, zynq7)
+N=60
```
Comment (litghost) [inline fuzzers/050-pip-seed/Makefile]: Should be "N ?= 60" to accommodate "QUICK".

Comment (tmichalak) [reply fuzzers/050-pip-seed/Makefile]: Sure, I will fix that right away.

**Comment by tmichalak** [comment]
> Same change to N variable, and LGTM

Which N variable do you have in mind here?

**Comment by litghost** [comment]
Why is this PR WIP?

File: fuzzers/050-pip-seed/Makefile (hunk 1 above)
Comment (litghost) [reply fuzzers/050-pip-seed/Makefile]: Still not fixed?

**Comment by tmichalak** [comment]
@litghost I removed the WIP and pushed (forgot to push the last time) the change related to one of you review comments. 
There is however one more comment of yours:

>Same change to N variable, and LGTM

Which N variable do you have in mind here?

**Comment by litghost** [comment]
> > Same change to N variable, and LGTM
> 
> Which N variable do you have in mind here?

https://github.com/SymbiFlow/prjxray/pull/837#discussion_r286535301

**Comment by tmichalak** [comment]
@litghost Hmm, okay, so you only meant one change. I fixed that particular line just few minutes ago. Not sure why kokoro is stuck?

**Comment by litghost** [comment]
> @litghost Hmm, okay, so you only meant one change. I fixed that particular line just few minutes ago. Not sure why kokoro is stuck?

Likely a problem on the github side, leave it for a bit.

**Comment by litghost** [comment]
Ya https://www.githubstatus.com/

**Pull request merged**